src/main/java/io/github/dft/kis/constantcode/ConstantCodes.java

```diff
@@ -0,0 +1,9 @@
+package io.github.dft.kis.constantcode;
+
+public interface ConstantCodes {
+    String CONTENT_TYPE = "Content-Type";
+    String ACCEPT = "Accept";
+    String HTTPS = "https://";
+    String BASE_ENDPOINT = "kis9.iconsignit.com.au/api/";
```

Comment:
This should be %s.iconsignit.com.au/api/

And while constructing SDK. Take another parameter to take clientDomain

Reply:
Added parameter while building the SDK for consuming clientDomain.